### PR TITLE
Feature/sujan week3

### PR DESCRIPTION
**Pull request by sujankanna470**

### CODE REVIEW

File: assignment/src/com/assignment3/nullpointer/NullException.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        }
+        try{
+            String name=null;
+            name.length();
+        }
+        catch(NullPointerException e)
+        {
+            System.out.print("End of main 2");
+        }
+    }
```
Comment (VBhaskarachary) [inline assignment/src/com/assignment3/nullpointer/NullException.java]: As discussed, Need one try for handling those both  exceptions

**Review by VBhaskarachary — COMMENTED** [review]
Please work on all reviewed comments which discussed in call also

File: assignment/src/com/assignment3/nullpointer/NullException.java (hunk 1 above)
Comment (sujankanna470) [reply assignment/src/com/assignment3/nullpointer/NullException.java]: I had made changes

**Review by VBhaskarachary — APPROVED** [review]
Changes looks good to me

**Pull request merged**